### DISCUSSION TITLE
Remix.run example?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [vidstack/player#1031] [Q&A]
Having a hard time getting vidstack integrated in remix. 
IS there an official doc example some guidance on how to proceed?

**Comment by mihar-22** [comment]
There's nothing that would really be different in the Remix example because it uses [Vite](https://vitejs.dev/) under the hood which our examples all use. If there's specific issues you're running into then open a help post on Discord and share information. Thanks :)

**Reply by Jarrodsz** [reply]
Hello Mihar thank you for the reply.
If i copy the exact config over from the next project to my remix app I always end up with a broken UI.

If i recreate the example in remix and put a minmal implementation repro on github would you mind take a peek?
It can then be added to the official repro example.

I think there are some remix quirks that you overlook and i ran into but a minimal repro would better illustrate that.

**Reply by mihar-22** [reply]
Ye of course I'll take a look, feel free to share a repro if there's a particular issue.
